### PR TITLE
added a native test for game stages

### PR DESCRIPTION
**Pull request by solebox**
added a native test for game stages

### CODE REVIEW

**Comment by solebox** [comment]
@vinsburg pull request incoming first naive test ;)

File: Cargo.toml [hunk 1]
```diff
@@ -6,3 +6,4 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+log = "0.4.22"
```
Comment (vinsburg) [inline Cargo.toml:9]: Logger seems unsued, please remove it.
I'm keeping the project dependency free for now.

File: src/game/mod.rs [hunk 2]
```diff
@@ -193,3 +193,19 @@ impl Game {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use log::debug;
```
Comment (vinsburg) [inline src/game/mod.rs:199]: `use log::debug;` Should be removed aswell.

File: src/game/stack/mod.rs [hunk 3]
```diff
@@ -6,6 +6,7 @@ use std::vec::IntoIter;
 
 use kind::{Kind, KindId};
 
+
```
Comment (vinsburg) [inline src/game/stack/mod.rs:9]: I've fixed spacing at a later commit, please delete  as well :pray:

**Review by vinsburg — CHANGES_REQUESTED** [review]
Awsome test, thanks for contributing!

**Comment by solebox** [comment]
oh dang i missed the comments, this pr gave me another idea

**Pull request merged**